test/kinematic/vector/velocity_of_transfer_between_reference_frames_test.py

```diff
@@ -0,0 +1,81 @@
+from collections import namedtuple
+from pytest import fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    Quantity,
+    QuantityVector,
+)
+from symplyphysics.core.approx import assert_equal_vectors
+from symplyphysics.laws.kinematic.vector import velocity_of_transfer_between_reference_frames as law
+
+Args = namedtuple("Args", "vtr v0 w r")
+
+
+@fixture(name="test_args")
+def test_args_fixture() -> Args:
+    vtr = QuantityVector([3, -3, 6], dimension=units.velocity)
```

Comment:
It's a bit harder to understand values this way. Using normal `meter / second` is easier to read.

Reply:
What's the point of the `dimension` parameter then? Also, shouldn't it be something like `measurement_unit` for quantities? It's somewhat counterintuitive to use dimension instead of units there. And I believe it's easier to understand this way, since there is much less clutter.

Reply:
No, I meant to write `3 * units.meter / units.second`

Reply:
I see, but isn't it repetitive (and error prone) to write it every time? We can factor it out into some `measurement_unit` parameter.

Reply:
Yes, a bit cluttered but helps to understand, what we are doing here. You can extract it to a separate variable, ie to `velocity_unit`